src/khoj/routers/api_chat.py

```diff
@@ -531,13 +540,24 @@ async def chat(
     region: Optional[str] = None,
     country: Optional[str] = None,
     timezone: Optional[str] = None,
+    image: Optional[ImageUploadObject] = None,
     rate_limiter_per_minute=Depends(
         ApiUserRateLimiter(requests=5, subscribed_requests=60, window=60, slug="chat_minute")
     ),
     rate_limiter_per_day=Depends(
         ApiUserRateLimiter(requests=5, subscribed_requests=600, window=60 * 60 * 24, slug="chat_day")
     ),
 ):
+    url = None
+    if image:
```

Comment:
Can we move this logic into `event_generator`? It'll consolidate the court a little bit.